test/capi/testBasic.cpp

```diff
@@ -0,0 +1,359 @@
+/*
+ * Copyright (c) 2020-2021 Samsung Electronics Co., Ltd. All rights reserved.
+
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+#include <thorvg_capi.h>
+#include "catch.hpp"
+
+TEST_CASE("Capi: Basic initialization", "[capiInitializer]")
+{
+    REQUIRE(tvg_engine_init(TVG_ENGINE_SW | TVG_ENGINE_GL, 0) == TVG_RESULT_SUCCESS);
+    REQUIRE(tvg_engine_term(TVG_ENGINE_SW | TVG_ENGINE_GL) == TVG_RESULT_SUCCESS);
+}
+
+TEST_CASE("Capi: Canvas initialization", "[capiCanvas]")
```

Comment:
I think it's a good idea to create a file for each test case group.
at least..
We classified CAPI into docs groups as shown below, so I think this classification can be referenced.
What do you think?

ThorVG_CAPI (BETA version) | ThorVG C language binding APIs
-- | --
Initializer | A module enabling initialization and termination of the TVG engines
SwCanvas | A module for rendering the graphical elements using the software engine
Paint | A module for managing graphical elements. It enables duplication, transformation and composition
Shape | A module for managing two-dimensional figures and their properties
Gradient | A module managing the gradient fill of objects
Picture | A module enabling to create and to load an image in one of the supported formats: svg, png and raw
Scene | A module managing the multiple paints as one group paint

Reply:
Refactored testBasic.c into four independent files